### DISCUSSION TITLE
How to install and on Win 11?

### DISCUSSION THREAD

**Discussion by rafikiphoto** [vzhd1701/evernote-backup#49] [General]
I downloaded the zip file and extracted it. Then I ran the .exe. I saw a terminal pop up briefly and then disappear Then nothing. The installation section here contains terms that mean nothing to me. I am keen to use this app but I need guidance on installation and use in Windows 11 in layman's terms. Am I missing some instructions somewhere?

**Comment by vzhd1701** [comment]
You have to run it using PowerShell or Command Prompt. See this [step-by-step guide](https://vzhd1701.notion.site/How-to-use-enex2notion-on-Windows-6fa980b489ab4414a5317e631e7f6bc6), but use `evernote-backup` instead of `enex2notion`.

**Reply by rafikiphoto** [reply]
Thank you!

**Comment by Munghorn** [comment]
hello, I hope this thread is still active. vzhd1701, thanks so much for the step by step guide. Would be hugely grateful for your help on one last point. Everything went smoothly until Step 8 - Enter .\enex2notion.exe --token YOUR_TOKEN_HERE notebook.enex to upload notebook.enex. I got the token from Notion ( via Connections tab, Internal Integration Secret). Am hoping it's just a syntax issue. Please clarify what the line punctuation should like with the token that starts "secretxxxxxxxxxxx. For example, do I delete the first "token" that appears in low case in your example; space or hyphen/s between end of token number and "notebook.enex, etc? Tried various things but got error messages, such as "Error: No such option:   " in Powershell.  And I swapped out all instances of  enex2notion with evernote-backup , including the download file as "bin_evernote_backup_1.9.3_win_x64 ". The attempt with your step by step guide comes after days of dead ends, as a non-techhie, and I really appreciated the clear instructions and the images. Again, would much appreciate your help to get over this Step 8 blockage. Many thanks, in advance!!
